tests/driver.C

```diff
@@ -134,15 +140,17 @@ int main(int argc, char ** argv)
   // Add all tests which match the re to the runner object.
   libMesh::out << "Will run the following tests:" << std::endl;
   const int n_tests_added =
-    add_matching_tests_to_runner(registry.makeTest(),
+    add_matching_tests_to_runner(suite,
                                  allow_regex_string, allow_regex,
                                  deny_regex_string, deny_regex,
                                  runner, rejects);
   if (n_tests_added >= 0)
     libMesh::out << "--- Running " << n_tests_added << " tests in total." << std::endl;
+  if (n_tests_added != -12345)
```

Comment:
Minor comment, but since this is our own magic number that we now actually have to refer to, it might make sense to use `libMesh::invalid_uint` or some other named constant.